src/base/QXmppGlobal.h

```diff
@@ -178,7 +178,9 @@ struct Success { };
 struct Cancelled { };
 
 ///
-/// Unit struct indicating a timeout (or keep-alive) error.
+/// Unit struct indicating a client-side timeout (or keep-alive) error.
+///
+/// It occurs if the own server does not respond within a given timeout.
```

Comment:
I suppose you suggested that because the error could occur when QXmpp is used as a server. Or are there other cases?We should maybe use the term `entity` instead of `peer` to be consistent since it is used at other places as well.

Reply:
I used peer to keep it generic for other connections we've got, like STUN/TURN. I'm also fine with entity.